tests/ui/app/reviews.js

```diff
@@ -132,7 +136,7 @@ casper.test.begin('Test review button hidden when logged in and cannot rate', {
             helpers.fake_login();
         });
 
-        helpers.waitForPageLoaded(function() {
+        waitForReviewsLoaded(function() {
```

Comment:
Login triggers a re-rendering of the views. Perhaps that's the issue?

![spinner-after-login mov](https://cloud.githubusercontent.com/assets/211578/6359616/107d8ccc-bc3a-11e4-8ed4-04d03832359c.gif)